evidence/create_package.go

```diff
@@ -95,7 +96,29 @@ func (c *createEvidencePackage) getPackageType(artifactoryClient artifactory.Art
 	return request.PackageType, nil
 }
 
-func (c *createEvidencePackage) getPackageVersionLeadArtifact(packageType string, metadataClient metadata.Manager) (string, error) {
+func (c *createEvidencePackage) getPackageVersionLeadArtifact(packageType string, metadataClient metadata.Manager, artifactoryClient artifactory.ArtifactoryServicesManager) (string, error) {
+	leadFileRequest := services.LeadFileParams{
+		PackageType:     strings.ToUpper(packageType),
+		PackageRepoName: c.packageRepoName,
+		PackageName:     c.packageName,
+		PackageVersion:  c.packageVersion,
+	}
+
+	leadArtifact, err := artifactoryClient.GetPackageLeadFile(leadFileRequest)
+	if err != nil {
+		log.Info("Trying to fetch LeadFile from metadata")
+		leadArtifactPath, err := c.getPackageVersionLeadArtifactFromMetaData(packageType, metadataClient)
+		if err != nil {
+			return "", err
+		}
+		return c.buildLeadArtifactPath(leadArtifactPath), err
```

Comment:
As the error is `nil` at this point (already checked), the `err` can be replaced to `nil`.